about.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Expo Book Fair | Capstone Portal Module</title>
+</head>
+<body>
+    <div class="container">
+        <header class="header">
+            <nav class="navbar" id="navbar">
+                <ul class="nav-up">
+                    <li><a href="#"><i class="fab fa-facebook"></i></a></li>
+                    <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">Mypage</a></li>
+                    <li><a href="#">Logout</a></li>
+                </ul>
+                <div class="nav-down">
+                    <a class="hamburger" href="#"><img  src="./Asset/hamburger-bar.png" alt="image for hamburger icon"></a>
+                    <a class="logo" href="#"><img src="./Asset/book-fair-logo.png" alt="Logo for expo book fair"></a>
+                    <ul class="nav-down-link">
+                        <li><a class="home-mob" href="index.html">Home</a></li>
+                        <li><a class="about-mob" href="about.html">About</a></li>
+                        <li><a class="close-btn" href="#"><i class="fas fa-times"></i></a></li>
+                    </ul>
+                </div>
+            </nav>
+            <div class="heading-container">
+                <img class="book-icons" src="./Asset/background/additional-background-overlay-icons.png" alt="background image icons">
+                <div class="heading">
```

Comment:
* Kindly stick to the original design as much as possible as your navbar and header section looks slightly different from the original design, please compare and make the necessary changes 🙏 
## ORIGINAL 
![image](https://user-images.githubusercontent.com/52497246/142419345-fd7b7111-b84b-4ddb-9c31-2b8c30881085.png)
## YOURS 
* Note it is required to stick to the Colors,
- Typography: font face, size and weight.
- Layout: composition and space between elements. Content can be different eg( text, title of the website etc)
![image](https://user-images.githubusercontent.com/52497246/142419436-d67fe298-a193-4302-9f33-2919352aed1f.png)